controllers/storagecluster/reconcile.go

```diff
@@ -526,6 +535,21 @@ func (obj *ocsCephConfig) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.S
 		return nil
 	}
 
+	if sc.Spec.Network.Provider == "" && sc.Spec.Network.DualStack {
+
+		networkConfig := &configv1.Network{ObjectMeta: metav1.ObjectMeta{Name: "cluster"}}
+		if err != nil {
+			return fmt.Errorf("could not get network config details to determine cloud platform: %v", err)
+		}
+		cidr1 := networkConfig.Spec.ClusterNetwork[0].CIDR
```

Comment:
let's loop through instead of assuming there are two

Reply:
Done